### ISSUE TITLE
[iOS] VoiceOver does not correctly describe View with GestureRecognizers

### ISSUE THREAD

**Issue by KillswitchPrime** [dotnet/maui#34380]
### Description

Taking something like a `VerticalStackLayout` and putting a `GestureRecognizer` on it, does NOT make VoiceOver say that the control is interactable. On Android, this works out of the box, with TalkBack saying: "Double tap to activate", but on iOS, VoiceOver says absolutely nothing.

This can be somewhat fixed by adding `SemanticProperties.Hint`. That causes VoiceOver to refer to the control as a "Button", but the downside is that all child elements will be ignored. So any labels inside the `VerticalStackLayout` are now completely ignored

### Steps to Reproduce

1. Turn on VoiceOver on iOS.
2. Open repro.
3. Verify that on the `VerticalStackLayout` without `SemanticProperties.Hint` there is no explanation that it is interactable.
4. Verify that on the `VerticalStackLayout` with `SemanticProperties.Hint`, all child `Labels` are ignored.

### Link to public reproduction project repository

https://github.com/KillswitchPrime/MauiGestureRecognizerIosA11y

### Version with bug

9.0.80 SR8

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

All iOS versions

### Did you find any workaround?

Adding `SemanticProperties.Hint="Some text"` does make VoiceOver describe the control as a "Button".
But all child elements are now ignored, so it becomes a lot more effort to make it work.

### Relevant log output

```shell

```

**Comment by MihuBot** [comment]
I'm a bot. Here is a possible related and/or duplicate issue (I may be wrong):
- https://github.com/dotnet/maui/issues/33612

**Comment by LogishaSelvarajSF4525** [comment]
This issue has been validated in Visual Studio Code 1.111.0 with MAUI versions 9.0.0, 9.0.80, 9.0.120,10.0.40 & 10.0.41. It can be reproducible on the iOS platform. Here I attached the video for reference:
In iOS (issue reproduced):
<video width="40" height="60" src="https://github.com/user-attachments/assets/fc973211-e5fc-482e-b35e-ddc4a805da44">

**Comment by Winnie-ZLR** [comment]
This issue still repro on 10.0.110-ci.inflight.26414.4

**Comment by Vignesh-SF3580** [comment]
> This issue still repro on 10.0.110-ci.inflight.26414.4

@Winnie-ZLR I have verified the issue in the latest MAUI version (10.0.100) and the latest inflight version (10.0.110-ci.inflight.26424.8), and I am unable to reproduce the issue in either version. When a gesture is added to a VerticalStackLayout with a semantic hint, it now reads the child control text along with the semantic hint/description. I have attached the sample and video for reference.

[MauiGestureRecognizerIosA11y-main.zip](https://github.com/user-attachments/files/31462662/MauiGestureRecognizerIosA11y-main.zip)

| MAUI version (10.0.100) | MAUI Inflight version (10.0.110-ci.inflight.26424.8) |
|----------|----------|
| <video width="300" height="600"  src="https://github.com/user-attachments/assets/6b18c6e3-0e27-4a22-a307-5840a2183649"> | <video width="300" height="600"  src="https://github.com/user-attachments/assets/37d8a0f3-53a9-478e-bdba-d31db71e688c"> |
